code/modules/projectiles/guns/energy.dm

```diff
@@ -65,11 +65,16 @@
 	readout += "Standard models of this projectile weapon have [span_warning("[ammo_type.len]")] mode\s"
 	readout += "Our heroic interns have shown that one can theoretically stay standing after..."
 	for(var/obj/item/ammo_casing/energy/for_ammo in ammo_type)
```

Comment:
If you could remove the type check here, using `as anything`, that'd be a code improvement.